e2e_playwright/conftest.py

```diff
@@ -241,6 +248,34 @@ def app(page: Page, app_port: int) -> Page:
     return page
 
 
+@pytest.fixture(scope="session")
```

Comment:
nit: Maybe you could do a quick try to move this as is into `st_camera_input_test.py`. I'm not sure if this works, but if it works it might be better to keep these configs only in `st_camera_input_test.py` to not cause any unexpected trouble with other tests.

Reply:
And you add a quick comment inside the function on why are are applying these changes.

Reply:
Unfortunately, we need to redefine arguments for browser launch context, which itself is a session-scoped fixture, and pytest reuses browser context for multiple tests. 
So we couldn't define it on the function/module level.

In case we really need to disable fake device stream for other tests (which shouldn't be the case, IMHO), we could run camera_input tests as separate CI job. 

I added explanatory comments to the fixture!